### PR TITLE
Fix issues in the merge and configuration scripts of IS and APIM accelerator modules on windows platform

### PR DESCRIPTION
**Pull request by NiTR0-CH4RG3R**
## Description

> Adding new `configure.ps1` and `merge.ps1` for windows, and removing unused `.bat` files

**Issue link:** 

- https://github.com/wso2/financial-open-banking/issues/87

**Doc Issue:** 

- https://github.com/wso2/docs-open-banking/issues/893

------

### Development Checklist

1. [ ] Build complete solution with pull request in place.
2. [ ] Ran checkstyle plugin with pull request in place.
3. [ ] Ran Findbugs plugin with pull request in place.
4. [ ] Ran FindSecurityBugs plugin and verified report.
5. [ ] Formatted code according to WSO2 code style.
6. [x] Have you verified the PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?
7. [x] Migration scripts written (if applicable).
8. [x] Have you followed secure coding standards in [WSO2 Secure Engineering Guidelines](http://wso2.com/technical-reports/wso2-secure-engineering-guidelines)?

### Testing Checklist

1. [ ] Written unit tests.
2. [ ] Verified tests in multiple database environments (if applicable).
3. [ ] Tested with BI enabled  (if applicable).

### CODE REVIEW

File: open-banking-accelerator/accelerators/ob-apim/bin/configure.ps1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
- # Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ # Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```
Comment (Ashi1993) [inline open-banking-accelerator/accelerators/ob-apim/bin/configure.ps1:1]: lets change the year to 2023-2024

File: open-banking-accelerator/accelerators/ob-apim/bin/merge.ps1 [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+# Copying all the new OB artifacts to the base product
+# Copy-Item -Force -Recurse -Verbose (Join-Path $WSO2_OB_ACCELERATOR_HOME "carbon-home\*") -Destination $WSO2_BASE_PRODUCT_HOME
+# Using Robocopy.exe becuase powershell Copy-Item cmdlet doesn't do recursive copying after a certain number of subdirectories.
+Write-Output "[INFO] Copying new open-banking artifacts..."
+Robocopy.exe (Join-Path $WSO2_OB_ACCELERATOR_HOME "carbon-home") $WSO2_BASE_PRODUCT_HOME * /E /NFL /NDL /NJH /NJS /nc /ns /np
+Write-Output "[INFO] All the new OB artifacts has been copied!"
+
+Write-Output "[INFO] Completed!"
+
+Exit-Clean
```
Comment (Ashi1993) [inline open-banking-accelerator/accelerators/ob-apim/bin/merge.ps1:82]: Add EoF line

File: open-banking-accelerator/accelerators/ob-is/bin/configure.ps1 [hunk 3]
```diff
@@ -1,4 +1,4 @@
- # Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ # Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```
Comment (Ashi1993) [inline open-banking-accelerator/accelerators/ob-is/bin/configure.ps1:1]: lets change the year to 2023-2024

File: open-banking-accelerator/accelerators/ob-is/bin/merge.ps1 [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    Remove-Item -LiteralPath $CONSENTMGR_RUNTIME_JS_BACKUP_PATH -Force
+}
+
+Remove-Item -LiteralPath $CONSENTMGR_PATH_ZIP -Force
+
+Get-ChildItem (Join-Path $WSO2_BASE_PRODUCT_HOME "repository\components\dropins") | Where-Object{$_.Name -Match "org.wso2.carbon.identity.application.authentication.handler.identifier-*"} | Remove-Item
+
+Write-Output "[INFO] Completed!"
+
+Exit-Clean
```
Comment (Ashi1993) [inline open-banking-accelerator/accelerators/ob-is/bin/merge.ps1:125]: Add EoF line

File: open-banking-accelerator/accelerators/ob-is/bin/configure.ps1 (hunk 3 above)
Comment (NiTR0-CH4RG3R) [reply open-banking-accelerator/accelerators/ob-is/bin/configure.ps1]: Done

File: open-banking-accelerator/accelerators/ob-is/bin/merge.ps1 (hunk 4 above)
Comment (NiTR0-CH4RG3R) [reply open-banking-accelerator/accelerators/ob-is/bin/merge.ps1]: Done

**Review by Ashi1993 — APPROVED** [review]

**Review by RivinduM — APPROVED** [review]

**Review by anjuchamantha — APPROVED** [review]

**Pull request merged**